scss/typography/_helpers.scss

```diff
@@ -38,6 +38,39 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+////
+/// @group typography-helpers-dropcaps
+////
+
+/// Default font size for dropcaps 
+/// @type Number
+$dropcaps-font-size: 5.20em !default;
+
+/// Default line height for dropcaps
+/// @type Number
+$dropcaps-line-height: 0.5 !default;
+
+/// Default vertically aligned position for dropcaps
+/// @type String
+$dropcaps-vertical-align: baseline !default;
+
+/// Default text transform for dropcaps
+/// @type String
+$dropcaps-text-transform: uppercase !default;
```

Comment:
The only `text-transform` values that could make sense in this component are:
- inherit
- uppercase

So we should rather provide a `$dropcaps-uppercase: true` Boolean here.